airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/conftest.py

```diff
@@ -166,3 +168,23 @@ def catalog_schemas_fixture(connector_config, docker_runner: ConnectorRunner, ca
             cached_schemas[stream.name] = stream.json_schema
 
     return cached_schemas
+
+
+@pytest.fixture
+def detailed_logger():
+    LOG_DIR = "acceptance_tests_logs"
+    if os.environ.get("ACCEPTANCE_TEST_DOCKER_CONTAINER"):
+        LOG_DIR = os.path.join("/test_input", LOG_DIR)
+    os.system(f"mkdir -p {LOG_DIR}")
```

Comment:
minor but it's generally advised to use subprocess (.run or .call) rather than os.system

Reply:
`Path.mkdir` will do this job